engines/saga/input.cpp

```diff
@@ -115,6 +115,7 @@ int SagaEngine::processInput() {
 			case Common::KEYCODE_F8:
 				break;
 #endif
+			case Common::KEYCODE_INVALID: // Quiet VS C4065 warning
```

Comment:
This looks more like a hack than anything else.
It would be better to move SAGA_DEBUG check outside the switch case and duplicate the processAscii line in the else clause of the ifdef

Reply:
Fixed in 11f60801273c189318de1239cfdf298773a4aa37